src/traffic_server/InkAPI.cc

```diff
@@ -9104,6 +9115,166 @@ TSSslContextDestroy(TSSslContext ctx)
   SSLReleaseContext(reinterpret_cast<SSL_CTX *>(ctx));
 }
 
+TSReturnCode
+TSSslClientCertUpdate(const char *cert_path, const char *key_path)
+{
```

Comment:
May want to make a worker function in iocore/net/SSL* land, so we don't have to pollute the InkAPI area with the SSL details.